templates/partials/answers/textfield.html

```diff
@@ -17,4 +17,21 @@
   "legendClasses": "u-vh",
   "mutuallyExclusive": mutuallyExclusive,
   "error": error
-}) }}
+} %}
+
+{% if answer.suggestions_url %}
+  {% set config = config | setAttribute("typeahead", {
+    "instructions": _("Use up and down keys to navigate suggestions once you\'ve typed more than two characters. Use the enter key to select a suggestion. Touch device users, explore by touch or with swipe gestures."),
+    "ariaYouHaveSelected": _("You have selected"),
+    "ariaMinChars": _("Enter 3 or more characters for suggestions."),
+    "ariaOneResult": _("There is one suggestion available."),
+    "ariaNResults": _("There are {n} suggestions available."),
+    "ariaLimitedResults": _("Results have been limited to 10 suggestions. Type more characters to improve your search."),
```

Comment:
We don't mark aria for translation in other places, can't remember if there was a specific reason for this. Worth checking.